src/index.ts

```diff
@@ -34,13 +37,17 @@ export default ({
   });
 
   useEffect(() => {
-    window.addEventListener('scroll', handle);
+    let scrollable = scrollingElement == null
```

Comment:
No need to compare to `null`.
`scrollingElement?.current ? scrollingElement.current : window ` should be sufficient

Reply:
OK, changing it....